src/Graby.php

```diff
@@ -263,6 +273,8 @@ private function doFetchContent($url)
 
         $html = $this->convert2Utf8($response['body'], $response['all_headers']);
 
+        $this->logger->debug('Fetched HTML:', ['html' => $html]);
```

Comment:
```suggestion
        $this->logger->debug('Fetched HTML', ['html' => $html]);
```

Reply:
Resolved in https://github.com/j0k3r/graby/pull/184/commits/0a7d2ca09e7a0f81d9d66ac24b4acad48990d09e